AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/README.md

```diff
@@ -53,55 +53,100 @@ You will need to download and install the following toolkits, tools, and compone
 Required AI Tools: <https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html>
 <br>If you have not already, select and install these Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.
 
+**2. (Offline Installer) Activate the AI Tools bundle base environment**
 
+If the default path is used during the installation of AI Tools:
+```
+source $HOME/intel/oneapi/intelpython/bin/activate
+```
+If a non-default path is used:
+```
+source <custom_path>/bin/activate
+```
+**3. (Offline Installer) Activate relevant Conda environment**
+
+```
+conda activate <offline-conda-env-name>
+``` 
+
+**4. Clone the GitHub repository**
+
+``` 
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/
+```
+
+**5. Install dependencies**
+``` 
+pip install -r requirements.txt
+pip install notebook
+```
 
 ## Run the Sample
 >**Note**: Before running the sample, make sure [Environment Setup](#environment-setup) is completed.
 Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
 * [AI Tools Offline Installer (Validated)](#ai-tools-offline-installer-validated)
 * [Docker](#docker)
 
-### AI Tools Offline Installer (Validated)  
-1. If you have not already done so, activate the AI Tools bundle base environment. If you used the default location to install AI Tools, open a terminal and type the following
+### AI Tools Offline Installer (Validated)
+
+**1. Register Conda kernel to Jupyter Notebook kernel**
+
+If the default path is used during the installation of AI Tools:
 ```
-source $HOME/intel/oneapi/intelpython/bin/activate
+$HOME/intel/oneapi/intelpython/envs/<offline-conda-env-name>/bin/python -m ipykernel install --user --name=<offline-conda-env-name>

```

Comment:
need to specify offline env name here, I suppose for this sample it should be pytorch-gpu
```suggestion
$HOME/intel/oneapi/intelpython/envs/pytorch-gpu/bin/python -m ipykernel install --user --name=pytorch-gpu

```